### PR TITLE
Fix failing files

### PR DESCRIPTION
**Pull request by ravishende**
Fixes almost all failing files in the linter. simpleMDE will still fail on the simpleMDE being undefined. However, once Sravya switches it to be downloaded in the repo and required in the JS rather than a script linking to the online version, it should go away.

### CODE REVIEW

File: src/dailyLog/simpleMDE.js
Comment (danwyl) [inline src/dailyLog/simpleMDE.js:1]: Why is it still unhappy with SimpleMDE?

**Review by danwyl — CHANGES_REQUESTED** [review]
We should find a way for the linter to be happy

File: src/dailyLog/simpleMDE.js
Comment (ravishende) [reply src/dailyLog/simpleMDE.js:1]: Will do this after 6:20 today. There are issues with functionality after putting simpleMDE in our repo, but until then I don't think we can import simpleMDE by name in the js file, which makes it undefined in the linter's view.

**Comment by ravishende** [comment]
We decided that pulling simpleMDE into the repo is too much work for very little benefit considering how much time we have left (Iron triangle). Thus, we are just adding SimpleMDE as a global variable in a comment to show the linter that it is coming from the html script tag.

**Review by danwyl — APPROVED** [review]

**Pull request merged**